opensubtitles/infos_test.go

```diff
@@ -0,0 +1,17 @@
+package opensubtitles
+
+import (
+	"testing"
+)
+
+func TestInfoService_Formats(t *testing.T) {
+	//TODO:
+}
+
+func TestInfoService_Languages(t *testing.T) {
```

Comment:
Codacy found an issue: [parameter 't' seems to be unused, consider removing or renaming it as _](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)